zeebe/gateway-protocol/src/main/proto/v2/audit-logs.yaml

```
@@ -0,0 +1,505 @@
+## Endpoint definitions
+paths:
+  /audit-logs/search:
+    post:
+      x-eventually-consistent: true
+      tags:
+        - Audit Log
+      operationId: searchAuditLogs
+      summary: Search audit logs
+      description: Search for audit logs based on given criteria.
+      requestBody:
+        required: false
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AuditLogSearchQueryRequest'
+      responses:
+        "200":
+          description: The audit logs search result.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AuditLogSearchQueryResult'
+        "400":
+          $ref: 'common-responses.yaml#/components/responses/InvalidData'
+        "401":
+          $ref: 'common-responses.yaml#/components/responses/Unauthorized'
+        "403":
+          $ref: 'common-responses.yaml#/components/responses/Forbidden'
+        "500":
+          description: An internal error occurred while processing the request.
+
+  /audit-logs/{auditLogKey}:
+    get:
+      x-eventually-consistent: true
+      tags:
+        - Audit Log
+      operationId: getAuditLog
+      summary: Get audit log
+      description: Get an audit log by auditLogKey.
```

🔧 We get a single entry
```suggestion
      description: Get an audit log entry by auditLogKey.
```